projects/ngrx.io/content/guide/store/testing.md

```diff
@@ -59,7 +61,126 @@ describe('Auth Guard', () => {
 });
 </code-example>
 
-### Using Store for Integration Testing
+### Example: Effect injecting state
+
+The mock store can simplify testing Effects that inject State using the RxJs `withLatestFrom` operator.  The example below shows the `addBookToCollectionSuccess$` effect displaying a different alert depending on the number of books in the collection state.
+
+<code-example header="collection.effects.ts">
+import { Injectable } from '@angular/core';
```

Comment:
The `<` and `>` need to be replaced with their HTML entities to display correctly in the snippet.

Reply:
Not sure what you mean.  On line 69?

Reply:
An example would be where it has `<fromBooks.State>` it should be `&lt;fromBooks.State&gt;`.

Reply:
Got it!  Fixed in 8b570ef.